specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -3813,6 +4077,48 @@
         }
       }
     },
+    "SslProtocolMatchConditionParameters": {
+      "description": "Defines the parameters for SslProtocol match conditions",
+      "required": [
+        "operator",
+        "typeName"
+      ],
+      "properties": {
+        "typeName": {
+          "type": "string",
+          "enum": [
+            "DeliveryRuleSslProtocolConditionParameters"
+          ]
+        },
+        "operator": {
+          "description": "Describes operator to be matched",
+          "type": "string",
+          "enum": [
+            "Equal"
+          ],
+          "x-ms-enum": {
+            "name": "SslProtocolOperator",
+            "modelAsString": true
+          }
+        },
+        "negateCondition": {
+          "description": "Describes if this is negate condition or not",
+          "type": "boolean"
+        },
+        "matchValues": {
+          "description": "The match value for the condition of the delivery rule",
+          "type": "array",
+          "items": {
+            "type": "string",
+		        "enum": [
```

Comment:
So only one of the 3 TLS values will be allowed in the array? Ex : [ "TLSv1.2" ]? If so, why not use a plain enum instead of an array? What about when new TLS versions are available?

Reply:
We allow users to set multiple TLS values in MatchValue array. AFD will check if any of the protocol matches that of the incoming request. 
We might support more protocols in the future, so I have updated it to an extensible string.